app/controllers/users/mfa_selection_controller.rb

```diff
@@ -10,13 +10,20 @@ class MfaSelectionController < ApplicationController
     def index
       two_factor_options_form
       @after_setup_path = after_mfa_setup_path
+      @sign_up_mfa_selection_order_bucket = AbTests::SIGN_UP_MFA_SELECTION.bucket(session.id)
```

Comment:
If we have a user here, should we consider using the user's UUID as the argument here? That way it'd be consistent between sessions.